src/containers/layouts/articles/article1/article1.component.tsx

```diff
@@ -53,7 +53,7 @@ class Article1Component extends React.Component<Article1Props> {
           </Text>
           <ArticleActivityBar
             style={themedStyle.activityContainer}
-            comments={article.comments}
+            comments={article.comments ? article.comments.length : 0}
```

Comment:
prefer creating variables for passing it to component props
like `commentsCount`

Reply:
Is there any purposes for this?

Reply:
cleaner jsx? :)

Reply:
Done.